corehq/apps/appstore/templates/appstore/project_info.html

```diff
@@ -3,11 +3,7 @@
 {% load i18n %}
 {% load compress %}
 
-{% block js %}{{ block.super }}
```

Comment:
Even though this uses `block.super`, the code in `facet_sidebar.js` is only for the sidebar and `appstore_base.js` is only for resizing the results list, so those modules don't need to be dependencies of `project_info.js`.